src/Components/DrillPage.js

```diff
@@ -33,32 +32,32 @@ componentDidUpdate() {
           </View>
           <TouchableOpacity
             style={styles.videoLink}
-            onPress={() => navigation.navigate('DrillAnimationPage', { drill: drill })}
+            onPress={() => navigation.navigate('DrillAnimationPage', { drill })}
           >
             <Text style={styles.videoLinkText}>Video</Text>
           </TouchableOpacity>
         </ImageBackground>
         <View style={styles.separator} />
+        <TouchableOpacity style={styles.favoriteContainer} onPress={() => this.props.toggleFavorite(drill)}>
```

Comment:
Destructure all the props at once and use the local variable afterwards, this makes it a lot easier to figure out which props are used in a component

Reply:
For @guillaumemoby You can destructure (since es5) like this:
```
const {
  favoriteContainer, 
  videoLinkText,
} = styles;
```
Then, you can access `favoriteContainer` as a const in your code

Reply:
@arribbar : I could need deeper explanation for this point

Reply:
Then, you call the correct style by doing simply a `<TouchableOpacity style={favoriteContainer} ... />`